packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -367,10 +367,9 @@ class _AndroidViewState extends State<AndroidView> {
       id: _id,
       viewType: widget.viewType,
       layoutDirection: _layoutDirection,
-      onPlatformViewCreated: widget.onPlatformViewCreated,
       creationParams: widget.creationParams,
       creationParamsCodec: widget.creationParamsCodec,
-    );
+    )..addOnPlatformViewCreatedListener(widget.onPlatformViewCreated);
```

Comment:
onPlatformViewCreated can be null here, we should only add it if it's non null (an assertion will fail  otherwise)

Reply:
Good catch!